### DISCUSSION TITLE
Drill-Down

### DISCUSSION THREAD

**Discussion by Vaelatern** [terrastruct/d2#239] [Feature Requests]
I'm enjoying D2 and am diagramming a large system with it.

Once thing I realize I'll need is the ability to go from one diagram to another one. Seeing a box that goes "Web Frontend" and being able to click it to see a diagram that describes that part.

In SVG, that could just be a link, and either I could give it an explicit URL or something clever could happen, but either way the outcome would be the same: click on the box to drill further down.

What do you all think?

**Comment by Vaelatern** [comment]
This note in `style.md` in the docs seems interesting and relevant.

```
# Or an internal board
link: Overview.Untitled Board 2
```

**Reply by alixander** [reply]
ah, this is another carry-over from when D2 was only running on the Terrastruct web app. You can't do it currently on local D2, we'll find a way to do so, but it's probably mediumish priority right now
